src/vmm/src/guest_config/templates/mod.rs

```diff
@@ -467,6 +467,12 @@ pub mod x86_64 {
         /// Failure in processing the MSRs in template for x86_64 CPU configuration.
         #[error("Template changes an MSR entry not supported by the host - [{0}]")]
         MsrNotSupported(String),
+        /// Failed to get KVM vCPU MSRs.
+        #[error("Failed to get KVM vCPU MSRs: {0}")]
+        VcpuGetMsrs(kvm_ioctls::Error),
+        /// The number of MSRs returned by the kernel is unexpected.
+        #[error("Unexpected number of MSRs reported by the kernel")]
+        VcpuGetMSRSIncomplete,
```

Comment:
nitpick: CamelCase
```suggestion
        VcpuGetMsrsIncomplete,
```

Reply:
ah, I see. I will fix this in commit 590a5ce518b2eec03888dc46c0c561b65179b549 of my PR #3586 !